### PR TITLE
make new mobile-menu

### PR DESCRIPTION
**Pull request by revengemiroz**

### CODE REVIEW

File: client/src/components/MobileSideMenu/index.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        title: 'Log Out',
+        to: 'log-out',
+        icon: <LogOut />,
+        onClick: () => handleLogout(),
+        exact: true,
+      },
+    ],
+  };
+
+  if (loading === false && !error) {
```
Comment (ivanms1) [inline client/src/components/MobileSideMenu/index.js]: you can make this just one if
```suggestion
  if (!loading && !error && currentUser?.role !== 'ADMIN') {
```

File: client/src/components/MobileSideMenu/style.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+  :hover {
+    transform: scale(1.02) !important;
+    transition: 0.1s linear !important;
+  }
+
+  :active {
+    transform: translateY(1px) scale(1.01) !important;
+  }
+  /* border: 2px solid green; */
```
Comment (ivanms1) [inline client/src/components/MobileSideMenu/style.js]: remove please

File: client/src/components/MobileSideMenu/style.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    color: #647380;
+    margin-top: 7px;
+  }
+`;
+
+export const Exit = styled.button`
+  background-color: transparent;
+  outline: none;
+  border: none;
+  /* border: 2px solid green; */
```
Comment (ivanms1) [inline client/src/components/MobileSideMenu/style.js]: remove

**Review by ivanms1 — CHANGES_REQUESTED** [review]
some small changes

File: client/src/components/MobileSideMenu/style.js (hunk 3 above)
Comment (revengemiroz) [reply client/src/components/MobileSideMenu/style.js]: done

File: client/src/components/MobileSideMenu/style.js (hunk 2 above)
Comment (revengemiroz) [reply client/src/components/MobileSideMenu/style.js]: done

File: client/src/components/MobileSideMenu/index.js (hunk 1 above)
Comment (revengemiroz) [reply client/src/components/MobileSideMenu/index.js]: done

**Review by ivanms1 — APPROVED** [review]
good job my son

**Pull request merged**